mail_tracking/models/mail_thread.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, api
+from email.utils import COMMASPACE
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    @api.multi
+    @api.returns('self', lambda value: value.id)
+    def message_post(self, body='', subject=None, message_type='notification',
+                     subtype=None, parent_id=False, attachments=None,
+                     content_subtype='html', **kwargs):
+        new_message = super().message_post(
```

Comment:
... and then this here:

```suggestion
        new_message = super().message_post(*args, **kwargs)
```

And it will do exactly the same.